plugins/feeds/public/virustotal_hunting.py

```diff
@@ -0,0 +1,62 @@
+import json
+from datetime import timedelta
+
+from core import Feed
+from core.observables import Hash, File
+from core.config.config import yeti_config
+import logging
+
+
+class VirusTotalHunting(Feed):
+
+    default_values = {
+        "frequency": timedelta(minutes=5),
+        "name": "VirusTotalHunting",
+        "source": "https://www.virustotal.com/intelligence/hunting/",
+        "description": "Feed of hunting of VirusTotal",
+
+    }
+
+    settings = {
+        'vt_url_hunting': {
+            'name': 'VT Url Hunting',
+            'description': 'on Virus Total you can make a feed in json or xml in tab hunting'
+        }
+    }
+
+    def update(self):
+        api_key = yeti_config.get('vt', 'key')
+
+        if api_key:
+            self.source = 'https://www.virustotal.com/intelligence/hunting/notifications-feed/?key=%s' % api_key
+            for item in self.update_json()['notifications']:
+                self.analyze(item)
+        else:
+            logging.error("Your VT API key is not set in the confile file")
+
+    def analyze(self, item):
+        tags = []
+        json_string = json.dumps(item)
+        context = {'source': self.name}
+
+        f_vt = File.get_or_create(value='FILE: {}'.format(item['sha256']))
```

Comment:
nit: remove space after the `:`, since this is just a way to distinguish the `File` observable from a `Hash` observable.

I note that this is bad practice on my behalf, where I should enforce this formatting somewhere else (ideally in the `File` observable constructor)